cmd/queue/main.go

```diff
@@ -146,6 +156,10 @@ func probeUserContainer() bool {
 // Make handler a closure for testing.
 func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, proxy *httputil.ReverseProxy) func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		recorder := pkghttp.NewResponseRecorder(w, http.StatusOK)
```

Comment:
Do we care to measure in the proxy span the probing part?

Reply:
How about 3 traces?  One for full length, one for the proxy, and one for the probe?

Reply:
That can work.